fabric/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,31 @@
+
+{
+  "schemaVersion": 1,
+  "id": "baritone",
```

Comment:
Can we maybe either use "Baritone" or "Baritoe" for all platforms instead of Forge -> "Baritoe" and Fabric -> "Baritone"?

Reply:
@leijurv why'd you let me merge fabric with baritone id back in 1.16
https://github.com/cabaletta/baritone/blob/1e7e504650222dc24e82379f1e81fbbb3c793e3e/src/launch/resources/fabric.mod.json#L4

and which should we use now

Reply:
> why'd you let me
LOL gonna stop you there, you're assuming I read your PRs??? hahaha

go flip a coin, you pick. baritone or baritoe

Reply:
```suggestion
  "id": "baritoe",
```